reference/pulseox-data-collector.v1.json

```diff
@@ -274,6 +411,9 @@
                       "Temperature"
                     ]
                   }
+                },
+                "": {
+                  "type": "string"
```

Comment:
This part looks like a mistake, can we remove it?

Reply:
Yes. My bad.